tests/normalize/test_model_item_normalizer.py

```diff
@@ -469,52 +469,54 @@ def test_dlt_column_addition_configs(
     if add_dlt_load_id and add_dlt_id:
         assert (
             normalized_query
-            == f'SELECT _dlt_subquery."a" AS a, _dlt_subquery."b" AS b, \'{load_id}\' AS'
-            " _dlt_load_id,"
-            " UUID() AS _dlt_id FROM (SELECT a, b FROM my_table) AS _dlt_subquery\n"
+            == f'SELECT _dlt_subquery."a" AS "a", _dlt_subquery."b" AS "b", \'{load_id}\' AS'
+            ' "_dlt_load_id", UUID() AS "_dlt_id" FROM (SELECT a, b FROM my_table) AS'
+            " _dlt_subquery\n"
         )
         assert (
             another_norm_query
-            == f'SELECT _dlt_subquery."a" AS a, _dlt_subquery."b" AS b, \'{another_load_id}\' AS'
-            ' _dlt_load_id, _dlt_subquery."_dlt_id" AS _dlt_id FROM (SELECT a, b,'
+            == 'SELECT _dlt_subquery."a" AS "a", _dlt_subquery."b" AS "b",'
+            f" '{another_load_id}' AS"
+            ' "_dlt_load_id", _dlt_subquery."_dlt_id" AS "_dlt_id" FROM (SELECT a, b,'
             " MY_CUSTOM_FUNC() AS _dlt_id FROM my_table) AS _dlt_subquery\n"
         )
     elif add_dlt_load_id and not add_dlt_id:
         assert (
             normalized_query
-            == f'SELECT _dlt_subquery."a" AS a, _dlt_subquery."b" AS b, \'{load_id}\' AS'
-            " _dlt_load_id"
+            == f'SELECT _dlt_subquery."a" AS "a", _dlt_subquery."b" AS "b", \'{load_id}\' AS '
+            '"_dlt_load_id"'
             " FROM (SELECT a, b FROM my_table) AS _dlt_subquery\n"
         )
         assert (
             another_norm_query
-            == f'SELECT _dlt_subquery."a" AS a, _dlt_subquery."b" AS b, \'{another_load_id}\' AS'
-            " _dlt_load_id FROM (SELECT a, b,"
-            " MY_CUSTOM_FUNC() AS _dlt_id FROM my_table) AS _dlt_subquery\n"
+            == 'SELECT _dlt_subquery."a" AS "a", _dlt_subquery."b" AS "b",'
+            f" '{another_load_id}' AS"
+            ' "_dlt_load_id" FROM (SELECT a, b, MY_CUSTOM_FUNC() AS _dlt_id FROM my_table) AS'
+            " _dlt_subquery\n"
         )
     elif not add_dlt_load_id and add_dlt_id:
         assert (
             normalized_query
-            == 'SELECT _dlt_subquery."a" AS a, _dlt_subquery."b" AS b, UUID() AS _dlt_id FROM'
+            == 'SELECT _dlt_subquery."a" AS "a", _dlt_subquery."b" AS "b", UUID() AS "_dlt_id" FROM'
```

Comment:
the model item normalizer will now always use quoted identifiers on the outer queries, regardless wether the inner select has quoted identifiers. Since all queries arriving here now go through query normalization in the dataset where quotes are applied, this should be fine acutally.